script.embuary.info/resources/settings.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+	<category label="128">
+        <setting label="TheMovieDB" type="lsep"/>
```

Comment:
Not mandatory but you could have some identation here for better readability. Last like is different.

Reply:
changed